runtimes/Dockerfile-ml

```diff
@@ -14,7 +14,8 @@ RUN pip install \
     numpy \
     pandas \
     scipy \
+    daiquiri \
```

Comment:
Note that we are changing the basic runtime here. Which is fine if daiquiri is the way to go. Just double checking.

Reply:
If the image should be generic enough ML image, it is probably not a good idea to include daiquiri in that case.